mean.hhs

```diff
@@ -0,0 +1,104 @@
+/**
+ * @author Jianan Lin (林家南)
+ * @param input - the list, matrix, tensor or structure that you want to get the mean(average) of
+ * @returns - a number representing the mean of all the elements of a list, matrix, tensor or other structure
+ * 
+ */
+
+
+*import math: is_number
+*import math: flatten
+*import math: ndim
+*import math: deep_copy
+
```

Comment:
needs to be in function now for all functions from now on, will update style guide